jdaviz/configs/specviz/tests/test_helper.py

```diff
@@ -248,7 +248,8 @@ def test_load_spectrum_list_directory(tmpdir, specviz_helper):
 
     with pytest.warns(UserWarning, match='SRCTYPE is missing or UNKNOWN in JWST x1d loader'):
         specviz_helper.load_spectrum(data_path)
-    assert len(specviz_helper.app.data_collection) == 3
+    # NOTE: the length was 3 before specutils 1.9 (https://github.com/astropy/specutils/pull/982)
+    assert len(specviz_helper.app.data_collection) == 40
```

Comment:
Should we also bump https://github.com/spacetelescope/jdaviz/blob/0b5a6beb21262fabbb2604cf0ee455240208289d/setup.cfg#L37 ? Then we don't need this comment. Otherwise, it is more correct to account for specutils version programmatically (either skip the test if specutils is too old or change the expected answer depending on specutils version).

Reply:
To recap my Slack post: 40 is expected if you count the following:

* 20 EXTRACT1D extensions from the first x1d in the ZIP file.
* 19 EXTRACT1D extensions from the second x1d in the ZIP file.
* 1 combined spectrum for all of them, generated during Specviz load time.

`20 + 19 + 1 = 40`

Reply:
I was wondering how to handle that... technically the CI we currently have setup will always pass, but a user with specutils < 1.9 that runs tests locally will see a failure here.  The other option would be to hardcode that logic in the test (to compare against 3 or 40 depending on the version of specutils)...

Reply:
> The other option would be to hardcode that logic in the test (to compare against 3 or 40 depending on the version of specutils)

That is my preferred solution if you don't mind the extra work. specutils version can be checked via `packaging.version.Version` or `astropy.utils.introspection.minversion`.